src/jv_aux.c

```diff
@@ -222,13 +222,17 @@ jv jv_dels(jv t, jv keys) {
     // no change
```

Comment:
I will make it static in a separate commit. `jv_delpaths` should be enough until someone specifically requests this in libjq for efficiency.